lib/web_ui/dev/licenses.dart

```diff
@@ -0,0 +1,96 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io' as io;
+
+import 'package:args/command_runner.dart';
+import 'package:path/path.dart' as path;
+
+import 'environment.dart';
+
+class LicensesCommand extends Command<bool> {
+  @override
+  final String name = 'licenses';
```

Comment:
How about "check-licenses"?

Reply:
Too long? :stuck_out_tongue:

Reply:
I imagine it will be used infrequently. Instead, we'll probably have a `felt presubmit` that we'll encourage everyone to run before sending a PR.

Reply:
I made the change anyway :)